docs/running-on-mesos.md

```diff
@@ -194,6 +194,10 @@ conf.set("spark.mesos.constraints", "tachyon:true;us-east-1:false")
 
 For example, Let's say `spark.mesos.constraints` is set to `tachyon:true;us-east-1:false`, then the resource offers will be checked to see if they meet both these constraints and only then will be accepted to start new executors.
 
+Spark rejects offers for slaves with unmet constraints for 120 seconds by default.
+This makes mesos not sending the rejected offers again and again.
+The reject offer duration is configurable using `conf.set("spark.mesos.rejectOfferDurationForUnmetConstraints", "60")` (for example).
```

Comment:
it doesn't make sense to document how to set this without documenting the configuration myself. I would just revert the changes here altogether since this seems too low level for the user to care about.